doc/changes/package_diffs/2.1.0/diff_standard-EXASOL-7.1.0_standard-EXASOL-7.1.0_without_python2.7/diff_flavor_base_deps_apt_get_packages.md

```diff
@@ -0,0 +1,113 @@
+|  46 | python-simplejson     | 3.13.2-1                        |                                 | REMOVED  |
+|  47 | python-sklearn        | 0.19.1-3                        |                                 | REMOVED  |
+|  48 | python-talloc         | 2.1.10-2ubuntu1                 |                                 | REMOVED  |
+|  49 | python3-boto          | 2.44.0-1ubuntu2.18.04.1         | 2.44.0-1ubuntu2.18.04.1         |          |
+|  50 | python3-cffi          | 1.11.5-1                        | 1.11.5-1                        |          |
+|  51 | python3-docutils      | 0.14+dfsg-3                     | 0.14+dfsg-3                     |          |
+|  52 | python3-jinja2        | 2.10-1ubuntu0.18.04.1           | 2.10-1ubuntu0.18.04.1           |          |
+|  53 | python3-ldap          | 3.0.0-1ubuntu0.1                | 3.0.0-1ubuntu0.1                |          |
+|  54 | python3-leveldb       | 0~svn68-3build3                 | 0~svn68-3build3                 |          |
+|  55 | python3-numpy         | 1:1.13.3-2ubuntu1               | 1:1.13.3-2ubuntu1               |          |
+|  56 | python3-openssl       | 17.5.0-1ubuntu1                 | 17.5.0-1ubuntu1                 |          |
+|  57 | python3-pandas        | 0.22.0-4ubuntu1                 | 0.22.0-4ubuntu1                 |          |
+|  58 | python3-paramiko      | 2.0.0-1ubuntu1.2                | 2.0.0-1ubuntu1.2                |          |
+|  59 | python3-pycparser     | 2.18-2                          | 2.18-2                          |          |
+|  60 | python3-pycurl        | 7.43.0.1-0.2                    | 7.43.0.1-0.2                    |          |
+|  61 | python3-pyftpdlib     | 1.5.1-4ubuntu1                  | 1.5.1-4ubuntu1                  |          |
+|  62 | python3-pygments      | 2.2.0+dfsg-1                    | 2.2.0+dfsg-1                    |          |
+|  63 | python3-pyodbc        | 4.0.17-1                        | 4.0.17-1                        |          |
+|  64 | python3-redis         | 2.10.6-2ubuntu1                 | 2.10.6-2ubuntu1                 |          |
+|  65 | python3-requests      | 2.18.4-2ubuntu0.1               | 2.18.4-2ubuntu0.1               |          |
+|  66 | python3-roman         | 2.0.0-3                         | 2.0.0-3                         |          |
+|  67 | python3-scipy         | 0.19.1-2ubuntu1                 | 0.19.1-2ubuntu1                 |          |
+|  68 | python3-setuptools    | 39.0.1-2                        | 39.0.1-2                        |          |
+|  69 | python3-simplejson    | 3.13.2-1                        | 3.13.2-1                        |          |
+|  70 | python3-sklearn       | 0.19.1-3                        | 0.19.1-3                        |          |
+|  71 | python3-smbc          | 1.0.15.6-1build2                |                                 | REMOVED  |
```

Comment:
We should mention this clearly in the release letter, because we're removing something besides python2

Reply:
Good point!